README.md

```diff
@@ -1,2 +1,13 @@
 In order to run server you need to use script runServer.sh (./runServer.sh).
-In order to run client there has to be currently running server of battleship game and then run runClient.sh script.
\ No newline at end of file
+In order to run client there has to be currently running server of battleship game and then run runClient.sh script.
```

Comment:
maybe retranslate to "In order to run client make sure that battleship game server is currently running and then run runClient.sh script"

Reply:
looks fine for me